tendermint/src/lite/types.rs

```diff
@@ -102,6 +102,7 @@ pub trait Vote {
     fn signature(&self) -> &[u8];
 }
 
+#[derive(Debug)]
```

Comment:
Didn't this change already make into `lite_impl_simple_merkle_merged` via https://github.com/interchainio/tendermint-rs/commit/9e00e99c2143f66a5dd61d80cb5c292a77eefbc6#diff-b1b6c7ff77dbf13838f76b8403715185?

Reply:
My mistake, just rebased this PR, only one line of change left.